net-irc/irssi/irssi-1.0.0.recipe

```diff
@@ -54,13 +52,18 @@ GLOBAL_WRITABLE_FILES="
 
 BUILD()
 {
+	export CFLAGS="-D_BSD_SOURCE"
+	export LDFLAGS="-lnetwork -lbsd"
```

Comment:
-lnetwork should be unnecessary (should be detected in configure).  What's needed from libbsd?